crates/red_knot_workspace/src/workspace.rs

```diff
@@ -301,13 +301,14 @@ impl Package {
     }
 
     /// Returns the files belonging to this package.
-    #[salsa::tracked]
```

Comment:
I'm not sure why I added caching for files when I implemented this. We only do the indexing once and then cache the result. There's no need to cache it again